HTI.md

```diff
@@ -400,7 +400,8 @@ This section provides an overview of the requirements and responsibilities in Mo
 | The  exchange of the message MUST NOT contain any information about: |
 | * the functional task, the definition of the task, and the people involved, and |
 | * information about the sending system, the recipient system and the message itself. |
-| The FHIR task object MUST use the STU3 version. |
+| The FHIR task object MUST at least use the STU3 version. |
+| The FHIR version MUST be set when not using the latest stable  FHIR version. |
```

Comment:
Ik zou expliciet maken welke FHIR versie de default is, en niet vertrouwen op "wat de meest recente versie" is. Een nieuwe versie van HTI kan dan kiezen een nieuwere versie van FHIR als default te nemen.

Reply:
Thanks voor de feedback JW! Voorzie je hier problemen mee? Ik heb dit namelijk juist niet benoemd om te voorkomen dat de documentatie op veel plekken aangepast moet worden. Ook wil ik graag voorkomen dat er bij een nieuwe FHIR versie een nieuwe versie van HTI gereleased moet worden.

Ik denk dat er altijd een afstemming tussen partijen plaats moet vinden over wanneer er overgegaan kan worden op een nieuwe FHIR release, ook als deze al officieel gereleased is en ook als we een nieuwe HTI versie maken.

Reply:
Ik denk dat het beter is expliciet te zijn, dat is het eigenlijk de enige reden. En inderdaad zal er altijd afstemming nodig zijn. Je zou nog verder kunnen gaan en zeggen: "de FHIR versie moet altijd expliciet benoemd worden. Voor backwards compatibility, als deze niet genoemd wordt, dan moet je aannemen dat het STU3 betreft."

Reply:
Direct genoemd aan het begin van het document, en een aparte sectie toegevoegd over de fhir-version